subprojects/docs/src/docs/userguide/what_is_gradle.adoc

```diff
@@ -12,133 +12,114 @@
 Extensibility::
-You can readily extend Gradle to provide your own task types or even build model. See the Android build support for an example of this: it adds many new build concepts such as flavors and build types.
+You can readily extend Gradle to provide your own build logic with custom tasks and plugins.
+See Android builds for an example: they add many new build concepts such as flavors and build types.
 IDE support::
-Several major IDEs allow you to import Gradle builds and interact with them: Android Studio, IntelliJ IDEA, Eclipse, and NetBeans. Gradle also has support for generating the solution files required to load a project into Visual Studio.
+Several major IDEs provide interaction with Gradle builds, including Android Studio, IntelliJ IDEA, Eclipse, VSCode, and NetBeans.
+Gradle can also generate the solution files required to load a project into Visual Studio.
 Insight::
-https://scans.gradle.com/[Build scans] provide extensive information about a build run that you can use to identify build issues. They are particularly good at helping you to identify problems with a build's performance. You can also share build scans with others, which is particularly useful if you need to ask for advice in fixing an issue with the build.
-
-[[five_things]]
-== Five things you need to know about Gradle
-
-Gradle is a flexible and powerful build tool that can easily feel intimidating when you first start. However, understanding the following core principles will make Gradle much more approachable and you will become adept with the tool before you know it.
+https://scans.gradle.com/[Build scans] provide extensive information about a build that you can use to identify issues.
+You can use build scans to identify problems with a build's performance, and even share them for debugging help.
 
-=== 1. Gradle is a general-purpose build tool
+== Concepts
 
-Gradle allows you to build any software, because it makes few assumptions about what you're trying to build or how it should be done. The most notable restriction is that dependency management currently only supports Maven- and Ivy-compatible repositories and the filesystem.
+It's helpful to know the following concepts before you dive into the details of Gradle.
 
-This doesn't mean you have to do a lot of work to create a build. Gradle makes it easy to build common types of project — say Java libraries — by adding a layer of conventions and prebuilt functionality through <<plugins#plugins,_plugins_>>. You can even create and publish custom plugins to encapsulate your own conventions and build functionality.
+=== Projects
 
-[[the_core_model_is_based_on_tasks]]
-=== 2. The core model is based on tasks
+Projects are the things that Gradle builds.
+Projects contain a build script, which is a file located in the project's root directory named `build.gradle` or `build.gradle.kts`.
```

Comment:
They can be named anything, but these are the default names.

Reply:
Is that detail necessary to bring up in the "What is Gradle" introduction? If so, I'd suggest this minor change to maintain technical correctness without too much extra verbiage:

```suggestion
Projects contain a build script, which is a file located in the project's root directory usually named `build.gradle` or `build.gradle.kts`.
```

Reply:
👍